src/deployers/PlatformPluginsDeployer.cpp

```diff
@@ -57,25 +57,26 @@ bool PlatformPluginsDeployer::deploy() {
         // either loading succeeds, then the system Gtk shall be used anyway, otherwise loading fails and Qt will fall
         // back to the default UI theme
         // we don't care whether this works (it's an experimental feature), therefore we ignore the return values
-        const auto libqgtk2Filename = "libqgtk2.so";
-        const auto libqgtk2styleFilename = "libqgtk2style.so";
-
-        const auto libqgtk2Path = platformThemesPath / libqgtk2Filename;
-        const auto libqgtk2stylePath = stylesPath / libqgtk2styleFilename;
-
-        // we need to check whether the files exist at least, otherwise the deferred deployment operation fails
-        if (bf::is_regular_file(libqgtk2Path)) {
-            ldLog() << "Attempting to deploy" << libqgtk2Filename << "found at path" << libqgtk2Path << std::endl;
-            appDir.deployFile(libqgtk2Path, platformThemesDestination);
-        } else {
-            ldLog() << "Could not find" << libqgtk2Filename << "on system, skipping deployment" << std::endl;
+        const auto libqgtk2Path = platformThemesPath / "libqgtk2.so";
```

Comment:
Why not put the filenames into const variables instead of hardcoding them in-line?

Reply:
Hm? What do you mean? They have to be hardcoded either way.

Reply:
Before, the filenames were separate variables.

But nevermind, it's really just a minor question of style. Let's merge this PR.